autopush/router/simple.py

```diff
@@ -98,11 +98,13 @@ def route_notification(self, notification, uaid_data):
             try:
                 result = yield self._send_notification(uaid, node_id,
                                                        notification)
-            except (ConnectError, UserError, ConnectionRefusedError):
+            except (ConnectError, UserError, ConnectionRefusedError), e:
                 self.metrics.increment("updates.client.host_gone")
                 dead_cache.put(node_key(node_id), True)
                 yield deferToThread(router.clear_node,
                                     uaid_data).addErrback(self._eat_db_err)
+                if type(e) is ConnectionRefusedError:
```

Comment:
nit: isinstance is generally preferred by all style guides, ie: https://github.com/amontalenti/elements-of-python-style#use-isinstanceobj-cls-not-typeobj--cls